Product_Landing_Page.html

```diff
@@ -44,6 +44,7 @@ <h1>This is a header</h1>
    </section>
     <footer>
         
+        
     </footer>
```

Comment:
There is no code inside this tag. I'll suggest deleting it or arranging some code inside it in order for it not to be redundant.

Reply:
Great job in mentioning the unused tag. Instead of writing in this way, I suggest writing it in a more simplified way. Like- Kindly note that there is no code between the footer tag. I would suggest you remove it completely for best HTML practices.